index.js

```diff
@@ -1,9 +1,33 @@
 'use strict';
-const urlRegex = require('url-regex');
+const urlRegex = require('url-regex')();
 const normalizeUrl = require('normalize-url');
+const queryString = require('query-string');
+const _ = require('underscore');
```

Comment:
I'm not interested in adding a huge dependency like this. You can easily do it without.

Reply:
done